src/Metric.php

```diff
@@ -92,6 +101,38 @@ public function scan($uri, \DOMXPath $xpath, $depth, Page $page, Metrics $contex
     }
 
     /**
+     * Get the machine name for a given error message
+     * 
+     * @param $error_message
+     * @return string
+     */
+    public function getMarkNameFromMessage($error_message)
+    {
+        switch ($error_message) {
```

Comment:
You are correct.  Do you think it would be better as a set of if/else statements, or do you think it would be better to change it to `switch(true)`?